### DISCUSSION TITLE
Need help with HTML/CSS for PnP Search Results Web Part

### DISCUSSION THREAD

**Discussion by egwrk** [microsoft-search/pnp-modern-search#3724] [Q&A]
I have a PnP Search Results Web Part and I'm trying to change the style so that it looks like the news web part with the tile or list layout.  

This is what I have so far by using the Card layout: 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/161059375/8bd706d2-1b07-48f6-b8ea-42b019a5aee0)

I removed a few fields so that only the news title shows:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/161059375/822526e5-e57e-43c5-a5a2-e8e5f86cefcb)
		

Is it possible to adjust the HTML/CSS of the results so that it has one of the layouts below. If so, can someone please share the code?

Tile layout:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/161059375/1361c23d-814e-44a5-947a-96b8aa10c2a6)

List layout: 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/161059375/21097851-261d-48f1-b37e-d40b3a7f3b43)

**Comment by kasperbolarsen** [comment]
When you have finalized the layout you are very welcome to add it to the Layouts repository, https://github.com/microsoft-search/pnp-modern-search-layouts/

**Comment by kasperbolarsen** [comment]
Have you had a look at the custom layout options here , https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/
In the custom layouts you can tweak the html and css as required
